core/src/main/java/nu/yona/server/messaging/service/DiscloseRequestMessageDTO.java

```diff
@@ -77,8 +71,9 @@ public Status getStatus()
 	public static DiscloseRequestMessageDTO createInstance(UserDTO actingUser, DiscloseRequestMessage messageEntity)
 	{
 		GoalConflictMessage targetGoalConflictMessage = messageEntity.getTargetGoalConflictMessage();
-		return new DiscloseRequestMessageDTO(messageEntity.getID(), messageEntity.getCreationTime(), messageEntity.getNickname(),
-				messageEntity.getStatus(), targetGoalConflictMessage.getOriginGoalConflictMessageID(),
+		return new DiscloseRequestMessageDTO(messageEntity.getID(), messageEntity.getCreationTime(), actingUser,
```

Comment:
That adds complexity and overhead for no reason. In this scenario, the user must still exist,  because they are sending a disclosure request.